catalog/rocks/entry.go

```diff
@@ -0,0 +1,45 @@
+package rocks
+
+import (
+	"crypto/sha256"
+
+	"github.com/treeverse/lakefs/graveler"
+	"google.golang.org/protobuf/proto"
+)
+
+var TombstoneValue graveler.Value
+
+func IsTombstoneValue(value graveler.Value) bool {
+	return len(value.Identity) == 0 && len(value.Data) == 0
+}
```

Comment:
rocks should consider about tombstones, It could use graveler.delete directly.
graveler.delete will create a tombstone in case it is needed